holoviews/operation/element.py

```diff
@@ -575,7 +575,7 @@ class decimate(ElementOperation):
        The x_range as a tuple of min and max y-value. Auto-ranges
        if set to None.""")
 
-    def _process(self, element, key=None):
+    def _apply(self, element, key=None):
```

Comment:
Yeah, I just noticed it [here](https://github.com/ioam/holoviews/blob/master/holoviews/core/operation.py#L116). Why aren't we using this method then?